### PR TITLE
Add Prometheus project

### PR DESCRIPTION
**Pull request by ivyostosh**
By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by ivyostosh** [comment]
/hold
Waiting for CDK changes and attribution file fixes

**Comment by ivyostosh** [comment]
/unhold
CDK changes and attribution file fixes are in place

File: projects/prometheus/prometheus/README.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+## **Prometheus**
+
```
Comment (TerryHowe) [inline projects/prometheus/prometheus/README.md:2]: I *thought* these readmes were supposed to have a version badge and build status, for example:

```
![Version](https://img.shields.io/badge/version-v0.1.1-blue)
![Build Status](https://codebuild.us-west-2.amazonaws.com/badges?uuid=eyJlbmNyeXB0ZWREYXRhIjoiN09rZ2FUVHBEMUZXeEV6VlBDLzEyNEVsN0FrU1d4U1pUUDQ5KzFlNGtyQjFVMitKRkRpSENRZzEyUUl3bjQwMUl3NElTQjc3T2krNFRaZGwvYWpKR0JvPSIsIml2UGFyYW1ldGVyU3BlYyI6InBvQ0lRWmVMODR4UG45NGMiLCJtYXRlcmlhbFNldFNlcmlhbCI6MX0%3D&branch=main)
```

File: projects/prometheus/prometheus/README.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+### Periodic Reviews
+Review [image releases](https://github.com/prometheus/prometheus/tags) periodically to identify new releases and decide on an update plan and an update schedule.
+
+### Updates
+
+#### Version changes
+1. Update the `GIT_TAG` and `HELM_GIT_TAG` files to have the new desired version based on the upstream release tags.
+1. Run `make generate` from the root of the repo to update the `UPSTREAM_PROJECTS.yaml` file.
+
```
Comment (TerryHowe) [inline projects/prometheus/prometheus/README.md:13]: No patches 🥇

**Review by TerryHowe — COMMENTED** [review]
Helm in a follow up or in another project directory?

File: projects/prometheus/prometheus/README.md (hunk 1 above)
Comment (ivyostosh) [reply projects/prometheus/prometheus/README.md:5]: Added

**Comment by ivyostosh** [comment]
> Helm in a follow up or in another project directory?

@TerryHowe Yes, trying to keep this one clean since it's getting quite complex already due to the build of the react app.

**Review by TerryHowe — APPROVED** [review]
/lgtm

**Comment by TerryHowe** [comment]
conflict

**Review by TerryHowe — APPROVED** [review]
/lgtm
/approve

**Pull request merged**